server/queries/gifQuery.js

```diff
@@ -1,10 +1,15 @@
 const createNewGif = 'INSERT INTO gifs (title, image_url, public_id, created_at, user_id) VALUES($1,$2,$3,$4, $5) RETURNING *';
-
+const selectGif = `SELECT * FROM gifs WHERE gif_id = $1`
```

Comment:
why did you not use this to select a single gif

Reply:
Ok sir. I have used it